src/hypergraph/hypergraph.cpp

```diff
@@ -559,6 +566,86 @@ void remove_free_nets(pmondriaan::hypergraph& H, size_t max_size) {
     }
 }
 
+/**
+ * Simplifies all duplicate nets.
+ */
+void simplify_duplicate_nets(pmondriaan::hypergraph& H) {

```

Comment:
Does not work on a distributed hypergraph right? As a net can locally look the same but not be the same globally.

Reply:
Yes, the simplify for parallel hypergraphs can be found in the main code of the partitioning along with the communication steps around it. This is for fully local hypergraphs.